### ISSUE TITLE
Weird lookup issue in NERVE (longer names?)

### ISSUE THREAD

**Issue by ilovan** [cwrc/CWRC-WriterBase#171]
https://github.com/ilovan/NERVE-integration-tests/blob/master/Salomania

![May-30-2019 16-59-35](https://user-images.githubusercontent.com/6600070/58670497-85d1cf00-82fc-11e9-87e9-72257c7cc6d3.gif)

**Comment by ajmacdonald** [comment]
https://github.com/cwrc/CWRC-WriterBase/commit/0beea6fcc767a535a4028cd62b1318de60af2b18

**Comment by ilovan** [comment]
@ajmacdonald , can you please check this out (I don't think it was ever implemented on any of the servers) . It's not working on cwrc.ca, cwrc-writer.cwrc.ca or dev-cwrc-writer.cwrc.ca

**Comment by ajmacdonald** [comment]
The above commit has been deployed so if that didn't fix the issue I'll need more explicit instructions about how to reproduce it.

**Comment by ajmacdonald** [comment]
https://github.com/cwrc/CWRC-WriterBase/commit/d4f6bf846f4551975d44f912b717e436ca65ea64

**Comment by ajmacdonald** [comment]
I can confirm that the first commit has in fact not been deployed on the integrated versions.

**Comment by ilovan** [comment]
@ajmacdonald , I don't think this was ever deployed, as the problem persists on dev-cwrc-writer

**Comment by ajmacdonald** [comment]
I can't reproduce it.

**Comment by ilovan** [comment]
maybe it's a mac vs pc thing. @lucaju, can you please see if you can reproduce it on your mac? 
I used this document for the screen capture: https://github.com/ilovan/NERVE-integration-tests/blob/master/Salomania

**Comment by lucaju** [comment]
@ilovan I can't reproduce the error as well.
This seems to be working fine. Even with extra spaces between "Margareth" and "Geertuida", entities lookup was able to produce results in different sources.

Perhaps this is an old issue, resolved in the last commits.
